### PR TITLE
Clarified keyword usage in call to Mie calculations

### PR DESCRIPTION
**Pull request by pcolarco**
Added argument names for Mie calculation call for clarification for optional keywords and added to CHANGELOG; 0 diff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### Changed - PRC, 2022-01-19
```
Comment (mathomp4) [inline CHANGELOG.md:8]: @pcolarco You don't need initials or anything here. Git and GitHub can tell us who changed a line of code.

Also, this should go under the `## [Unreleased]` section of the changelog.

Would you like me to fix this up for you? GitHub is showing a conflict anyway in the changelog that needs to be resolved. (I think your local `develop` might be a bit out of date?)

Comment (pcolarco) [reply CHANGELOG.md:8]: @mathomp4 Could you please fix this?
I got with @mmanyin to commit the changes to my branch. We tried to go further and issue a pull request, but you are correct I'm on an older model (10.17.6). I have no idea how to merge this into a newer model and thought I was waiting for someone to bless GOCART2G in the newer model but made this fix anyway. Sorry, don't really know what I'm doing here.

Comment (mathomp4) [reply CHANGELOG.md:8]: Oh no, it should work. Let me try...

**Comment by pcolarco** [comment]
Here are the things I don't know how to do:
1) compare my feature branch to whatever is the latest version of the repository. The names are arcane and I don't know if I should look at something chronological. I know Arlindo made changes at least to RC files under @GOCART, but I don't even know what version he's on
2) try any of this out in a recent model. GOCART2G is not present in @GOCART in my attempt to check out latest, but that's probably a main tag and maybe I need something else? No idea.
Mostly I'm just ignorant.

**Review by adarmenov — DISMISSED** [review]

**Comment by amdasilva** [comment]
Mie code being refactored elsewhere to clean up class names, methods, etc.

**Review by amdasilva — COMMENTED** [review]
Looks right, I guess. Thgis code is being revamped, not too concerned.

**Pull request merged**